otter/integration/tests/test_convergence.py

```diff
@@ -993,6 +993,47 @@ def test_false_negative_on_server_create_from_nova_no_overshoot(self):
                       matcher=HasLength(2), timeout=600)
         return d
 
+    @skip_me("Skipped until this error transition is fixed in Otter #1510")
+    @skip_if(not_mimic, "This requires Mimic for error injection.")
+    @tag("CATC-024")
+    @inlineCallbacks
+    def test_various_nova_40xs(self):
```

Comment:
Since this is only 1 400 case, should the test name be `test_nova_bad_request` or something similar?